crm_compassion/model/event_compassion.py

```diff
@@ -13,6 +13,14 @@
 from datetime import datetime
 
 
+class CompassionChild(models.Model):
+    _inherit = 'compassion.child'
+
+    @api.multi
+    def child_available(self):
+        self.state = 'D'

```

Comment:
If you do like this, all new children will be delegated which is not what we want. We want only children taken for events that become delegated.